site/quickstart.md

```diff
@@ -1,159 +1,171 @@
 # Quick start guide
 
-Use this quick start guide to create a WebLogic deployment in a Kubernetes cluster.
+Use this quick start guide to create a WebLogic deployment in a Kubernetes cluster with the Oracle WebLogic Kubernetes Operator. Please note that this walk-through is for demonstration purposes only, not for use in production.
 
-## 1.	Get the images and put them into your local registry.
+## Prerequisite
+For this exercise, you’ll need a Kubernetes cluster. If you need help setting one up, check out our [cheat sheet](k8s_setup.md).
 
-For the Operator image:
+## 1.	Get these images and put them into your local registry.
+
+a.	Pull the operator image:
 ```
 $ docker pull oracle/weblogic-kubernetes-operator:2.0
 ```
-For the Traefik image:
+b.	Pull the Traefik load balancer image:
 ```
 $ docker pull traefik:latest
 ```
+c.	Pull the WebLogic 12.2.1.3 install image:
```

Comment:
Don't we also need a step for building a patched 12.2.1.3 weblogic image?

Reply:
Maybe. I'm hoping that you or Monica gives me these specifics (as I don't know them).

Reply:
I added it (from Monica).